app/views/admin/loans/timeline/_main.slim

```diff
@@ -19,4 +19,9 @@ section.timeline-table
     li: a href="#" data-action="add-child-group" = t(".add_child_group")
     li: a href="#" data-action="add-child-step" = t(".add_child_step")
     li: a href="#" data-action="edit" = t(:edit)
-    li: a href="#" data-action="delete" = t(:delete)
\ No newline at end of file
+    li.disabled: a href="#" Delete
```

Comment:
Please add a comment explaining why there are two versions here and how they get shown/hidden (via the with-children class), etc. Else this is kind of cryptic.